utils/global-state-update-gen/src/main.rs

```diff
@@ -0,0 +1,108 @@
+use casper_engine_test_support::internal::LmdbWasmTestBuilder;
+use casper_execution_engine::shared::stored_value::StoredValue;
+use casper_types::{bytesrepr::ToBytes, Key};
+
+use clap::{App, Arg};
+
+mod auction_utils;
+mod utils;
+
+use crate::{
+    auction_utils::{
+        gen_snapshot, generate_entries_removing_bids, generate_entries_removing_withdraws,
+    },
+    utils::{hash_from_str, validators_diff},
+};
+
+/// Prints a global state update entry in a format ready for inclusion in a TOML file.
+fn print_entry(key: &Key, value: &StoredValue) {
+    println!("[[entries]]");
+    println!("key = \"{}\"", key.to_formatted_string());
+    println!("value = \"{}\"", base64::encode(value.to_bytes().unwrap()));
+    println!();
+}
+
+fn main() {
```

Comment:
This binary can do only one thing, right? It can only replace the validators?

Reply:
Right now, yes. It will probably gain other features in the future, though.